crates/cli/src/utils/cmd.rs

```diff
@@ -414,3 +414,25 @@ pub async fn print_traces(result: &mut TraceResult, decoder: &CallTraceDecoder)
     println!("Gas used: {}", result.gas_used);
     Ok(())
 }
+
+pub fn generate_local_signatures(output: &ProjectCompileOutput, cache_path: PathBuf) -> Result<()> {
```

Comment:
this needs docs and should be renamed to cache_local_signatures

Reply:
ok i'll fix it later